fhevm/tfhe.go

```diff
@@ -18,9 +18,11 @@ package fhevm
 
 /*
 #cgo linux CFLAGS: -O3 -I../tfhe-rs/target/release
-#cgo linux LDFLAGS: -L../tfhe-rs/target/release -l:libtfhe.a -lm
+#cgo linux CFLAGS: -O3 -I../tfhe-rs/target/release/deps
```

Comment:
You can merge this line with the previous to have a single `-O3` and the two includes